### PR TITLE
adding pegout documentation about how to use Metamask and Liquality

### PR DESCRIPTION
**Pull request by alexjavabraz**
## What

- added a pegout section
- added a documentation about how to create pegout using Liquality
- added a documentation about how to create pegout using Meta Mask

## Why

- the version 1.3.0 with implements pegout will be deployed in prod soon
- (if this patch addresses and existing, feel free to copy this section across)

## Refs

- (optional: include links to other issues, PRs, tickets, etc)

### CODE REVIEW

File: assets/img/guides/two-way-peg-app/tx-status.png
Comment (josedahlquist) [inline assets/img/guides/two-way-peg-app/tx-status.png:1]: can you please update this image to the latest version?

**Review by josedahlquist — APPROVED** [review]
other than my comment, LGTM

File: assets/img/guides/two-way-peg-app/tx-status.png
Comment (alexjavabraz) [reply assets/img/guides/two-way-peg-app/tx-status.png:1]: updated

**Review by josedahlquist — APPROVED** [review]

**Review by owans — APPROVED** [review]
Added page to left nav and minor fixes to layout.

**Pull request merged**